components/General/Faram/validations.js

```diff
@@ -175,7 +170,7 @@ const TIME_SEPARATOR = ':';
 export const timeCondition = (value) => {
     let error;
 
-    if (value !== undefined) {
+    if (isTruthy(value)) {
```

Comment:
Shouldn't this be similar to that of `dateCondition` i.e. `isTruthy(value, [''])`